server/views/pages/bookAVisit/visitors.njk

```diff
@@ -194,6 +228,8 @@
             attributes: { "data-test": "submit" },
             preventDoubleClick: true
           }) }}
+
+          <a href="/back-to-start" class="line-height-a-tag govuk-!-padding-left-4 govuk-link--no-visited-state">Start again</a>
```

Comment:
This isn't quite styled right (text is too small and not vertically centred). I think it should sort itself out if you use the pattern for button + link in the design system. See https://design-system.service.gov.uk/components/button/#grouping-buttons. There's a class to wrap the button and link in using a `<div>`. Then you'll no longer need the new `line-height-a-tag` class.